homeassistant/components/whois/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "whois",
   "name": "Whois",
   "documentation": "https://www.home-assistant.io/integrations/whois",
-  "requirements": ["python-whois==0.7.3"],
+  "requirements": ["whois==0.9.13"],
```

Comment:
I browsed the repo a bit.

https://github.com/DannyCork/python-whois/

There's a 0.9.14 release on github but it's not on PyPI. The setup.py file has a stale version spec.

There are some print statements but they won't be run in the normal code flow for us, only on error.

Reply:
Hmm the stale version will actually cause issues for the wheels build.

Will open a PR upstream.

Reply:
Instead of a PR, I've opened up an issue upstream explaining the difference (as there are multiple different things going on there).

As for the wheels, this isn't an issue at this point, as the publish package on PyPi has the correct version for this PR (0.9.13, which is the source of our wheels building).

Reply:
Marked the PR ready for review again, as I think we can move forward without awaiting the upstream change.